### PR TITLE
Added Bablic App

### PR DESCRIPTION
**Pull request by IshaiJaffe**
Bablic is SaaS that manages localization of a website and integrates using Javascript.

### CODE REVIEW

**Comment by jvoisin** [comment]
Your commit doesn't add any way to actually __detect__ the application :/

File: src/apps.json [hunk 1]
```diff
@@ -828,6 +828,14 @@
 			},
 			"website": "http://bigace.de"
 		},
+		"Bablic": {
+		      "cats": [
+				"3","9"
+			],
+			"website":"https://www.bablic.com/",
+			"env":"bablic",
```
Comment (jvoisin) [inline src/apps.json]: What about using `^bablic$` instead?

Comment (IshaiJaffe) [inline src/apps.json]: OK.
It should qualify is window.bablic is defined. Right?

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**